.claude/skills/cannbot-skills/ascendc-kernel-develop-workflow/scripts/verify_cmake_config.py

```
@@ -0,0 +1,139 @@
+        content = f.read()
+    
+    errors = []
+    warnings = []
+    
+    # 检查1：必须包含 find_package(ASC REQUIRED)
+    if 'find_package(ASC REQUIRED)' not in content:
+        errors.append("❌ 缺少 find_package(ASC REQUIRED)")
+    
+    # 检查2：project 必须包含 ASC 语言
+    if not re.search(r'project\s*\([^)]*LANGUAGES[^)]*ASC', content, re.IGNORECASE):
+        errors.append("❌ project() 必须包含 LANGUAGES ASC CXX（当前缺少 ASC）")
```

Ruff (0.15.7)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)